s30904-JakubTreder/zajecia3/zadanie4_zajecia3.txt

```diff
@@ -1,9 +1,22 @@
 1. 
+komenda bg sluzy do wznowienia w tle wstrzymanego programu.
+wykoanie:
 -bg 
+wniosek: program został wznowiony
+
+komenda fg słuzy do wznowienia na pierwszym planie wstrzymanego procesu.
+wykonanie:
 -fg 
+wniosek: program został wznowiony
 
+Wykoananie kolejnego zadania:
 -gimp &
 -gedit &
 -geany &
+wynik: zastały otwarte trzy programy terminal czeka na kolejne polecenia.
+
+
+Przywróć program gedit (lub gimp, w każdym razie nie uruchomiony jako ostatni program) z tła na pierwszy plan. 
+Skorzystaj z jobs aby dowiedzieć się jakie mają numery poszczególne zadania działające w tle.
 
-fg%(3605026)
+wykonanie: fg%(3605026)
```

Comment:
Dziwnie to wygląda. Naprawdę masz uruchomione przez siebie w terminalu 3 miliony 605 tysięcy 26 aplikacji w tle?